vortexasdk/endpoints/eia_forecasts.py

```diff
@@ -0,0 +1,80 @@
+"""EIA Forecasts Endpoint."""
+
+from vortexasdk.endpoints.eia_forecasts_result import EIAForecastResult
+from vortexasdk.endpoints.endpoints import EIA_FORECASTS_RESOURCE
+from vortexasdk.operations import Search
+from vortexasdk.api.shared_types import to_ISODate
+
+
+class EIAForecasts(Search):
+    """EIA forecasts Endpoint, use this to search through Vortexa's EIA Forecasts data.
+
+    The data includes:
+
+    - Vortexa's data science based forecast of the EIA number to be published on the week
+    - Actual EIA import/export numbers as published by the EIA Weekly Supply Estimates report
+    - EIA stocks (kbl)
+    - Cover (days of Supply for the whole of the US, as published by the EIA Weekly Supply Estimates report)
+    - Refinery runs (refiner “Percent Operable Utilization” as published by the EIA Weekly Supply Estimates report)
```

Comment:
Nice - should we make this even clearer by specifying the column name first, then the accompanying documentation? something like:

```runs: Refinery runs (refiner...```

this means it'll be easier for clients to lookup the functionality associated with a col name